core/types/rollup_cost_test.go

```diff
@@ -206,6 +208,40 @@ func TestExtractFjordGasParams(t *testing.T) {
 	require.Equal(t, fjordFee, c)
 }
 
+func TestExtractIsthmusGasParams(t *testing.T) {
+	zeroTime := uint64(0)
+	// create a config where holocene is active
```

Comment:
nit
```suggestion
	// create a config where isthmus is active
```

Reply:
@leruaa would you mind accepting this commit suggestion?